Assets/Scripts/Input/Player.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Script controls player character behaviour
+/// </summary>
+public class Player : MonoBehaviour
+{
+    #region Fields
+
+    public float runningVelocity;
+    public float fallVelocityLimit;
+    public float jumpVelocity;
+
+    public float yAccel;
+    private float yVelo = 0f;
```

Comment:
Maybe rename to yVelocity so have all variables uniformly named, and not some named velocity and some named velo
```suggestion
    private float yVelocity = 0f;
```